2021-04-19 05:50:30,882 DEBUG    root            Loaded Command Group: ['gcloud', 'auth']
2021-04-19 05:50:30,885 DEBUG    root            Loaded Command Group: ['gcloud', 'auth', 'docker_helper']
2021-04-19 05:50:30,888 DEBUG    root            Running [gcloud.auth.docker-helper] with arguments: [METHOD: "get"]
2021-04-19 05:50:30,906 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true
2021-04-19 05:50:30,910 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2021-04-19 05:50:30,913 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true HTTP/1.1" 200 None
2021-04-19 05:50:30,915 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token
2021-04-19 05:50:30,917 DEBUG    urllib3.connectionpool Resetting dropped connection: metadata.google.internal
2021-04-19 05:50:30,920 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token HTTP/1.1" 200 None
2021-04-19 05:50:30,926 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true
2021-04-19 05:50:30,928 DEBUG    urllib3.connectionpool Starting new HTTP connection (1): metadata.google.internal:80
2021-04-19 05:50:30,932 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/?recursive=true HTTP/1.1" 200 None
2021-04-19 05:50:30,933 DEBUG    google.auth.transport.requests Making request: GET http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token?scopes=%0Aemail%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fbigquery%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdevstorage.full_control%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fndev.cloudman%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fsqlservice.admin%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fprediction%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fprojecthosting%0A
2021-04-19 05:50:30,937 DEBUG    urllib3.connectionpool Resetting dropped connection: metadata.google.internal
2021-04-19 05:50:30,940 DEBUG    urllib3.connectionpool http://metadata.google.internal:80 "GET /computeMetadata/v1/instance/service-accounts/senmioxl@gmail.com/token?scopes=%0Aemail%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fappengine.admin%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fbigquery%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcompute%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fdevstorage.full_control%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fndev.cloudman%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fsqlservice.admin%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fprediction%0Ahttps%3A%2F%2Fwww.googleapis.com%2Fauth%2Fprojecthosting%0A HTTP/1.1" 200 None
2021-04-19 05:50:30,945 INFO     root            Display format: "json"
2021-04-19 05:50:31,016 INFO     ___FILE_ONLY___ {
  "Secret": "ya29.a0AfH6SMBXpuXaFjBk99k89VsEGUATHKuHVoPE2cj3hztnLG4UEtItFealdN5oOnH5fCR1U51_PuANRGLTz1-Y16Cc8sl4ysjuYd4J-WZYcUTuWaiPGj52dRmGDTAxTsMzrtzH7d5MpICdbkFzuPaNfY1tbyYjXKlmg5qC-JBC7qfBp9GKK1bfiO7XDHNJtq48MnmweG5WCbBTIVxyFwDRSSOlnOO2tlQrV2XXZyv5i7oOZLQQCpVMVjoGZr3FckTv46EkO_0",
  "Username": "_dcgcloud_token"
}
2021-04-19 05:50:31,016 INFO     ___FILE_ONLY___ 

2021-04-19 05:50:31,016 DEBUG    root            SDK update checks are disabled.
